packages/api/lambdas/sf-event-sqs-to-db-records/write-granules.js

```diff
@@ -165,7 +165,8 @@ const _writeFiles = async ({
   filePgModel = new FilePgModel(),
 }) => pMap(
   fileRecords,
-  async (fileRecord) => filePgModel.upsert(knex, fileRecord)
+  async (fileRecord) => filePgModel.upsert(knex, fileRecord),
+  { stopOnError: false }
```

Comment:
This is interesting. It was actually doing what we wanted without this line (the default is `true`). Making this flag `false` gets us a more useful `AggregateError` though. 

Either way it does all of the upserts even if one fails in the middle.